docs/source/zero-code-change.rst

```diff
@@ -0,0 +1,209 @@
+cuml.accel: Zero Code Change Acceleration for Scikit-Learn, UMAP and HDBSCAN
+============================================================================
+
+Starting in RAPIDS 25.02.01, cuML offers a new way to accelerate existing code
+based on Scikit-Learn, UMAP-Learn, and HDBSCAN. Instead of rewriting that code
+to import equivalent cuML functionality, simply invoke your existing,
+unaltered Python script as follows, and cuML will accelerate as much of the
+code as possible with NVIDIA GPUs, falling back to CPU where necessary:
+
+.. code-block::
+
+    python -m cuml.accel unchanged_script.py
+
+The same functionality is available in Jupyter notebooks using the
+following magic at the beginning of the notebook (before other imports):
+
+.. code-block::
+
+   %load_ext cuml.accel
+   import sklearn
+
+You can see an example of this in
+`KMeans Digits Notebook <zero_code_change_examples/plot_kmeans_digits.ipynb>`_, where an unmodified
+Scikit-Learn example notebook is used to demonstrate how ``cuml.accel`` can be
+used in Jupyter.
+
+In any Python environment, the following code snippet can also be used to
+activate ``cuml.accel`` if it is run prior to importing the module you wish to
+accelerate:
+
+.. code-block::
+   from cuml.accel import install
+   install()
+   import sklearn
```

Comment:
Looks like there might be an issue with the snippet. It isn't visible on Github.